.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "lib/libQtOlm"]
+	path = lib/libQtOlm
+	url = git@gitlab.com:aa13q/libqtolm.git
```

Comment:
agree, moved to upstream project: https://gitlab.com/b0/libqtolm/merge_requests/2

Reply:
Temporary switched back to get Travis CI working (using my olm fork with my libQtOlm fork)

Going to upstream somehow [proposed olm patch](https://github.com/a-andreyev/olm/commit/5421c05f86e6936ccfb381d263fe35aab0b230da)
And then provide new PR to BlackHat with fixed matrix-org git submodule. My current libQtOlm fork is [here](https://gitlab.com/aa13q/libqtolm)

Reply:
JFYI: hosting olm at GitHub is generally not advised - it contains cryptography and formally can be governed by all kinds of weird regulations about its export etc. While your patch is being reviewed, you can clone olm from the canonical repo and patch it locally (inside Travis box, in particular).

And, given that I don't see olm submodule in the original QtOlm repo, maybe align with @encombhat upfront. That has nothing to do with export restrictions, just a question of preferences how to handle deps.

Reply:
Thank you for the hints!
Added MR without submodule currently (since we could add them manually too via CI anyway) https://gitlab.com/b0/libqtolm/merge_requests/3
@encombhat, I'm ready to add MR with olm submodule to libQtOlm later if you decide we need it, but looks like `CMakeLists.txt` changes is enough for now :slightly_smiling_face:

Reply:
Switched back to b0/libQtOlm repo! :)
But probably need help with applying [the patch](https://github.com/a-andreyev/olm/commit/5421c05f86e6936ccfb381d263fe35aab0b230da.patch) to the olm repo via Travis CI (saved for later as side task)

Reply:
No need to patch olm! :slightly_smiling_face: :tada: https://gitlab.matrix.org/matrix-org/olm/commit/aa0c9ab6b51d182cb78ab4cc75c81c0054765bde
Travis CI is currently failing at another part (`qmc-example`) though :thinking: